service/resource/configmap/resource.go

```diff
@@ -230,7 +229,7 @@ func (r *Resource) GetUpdateState(ctx context.Context, obj, currentState, desire
 	}
 
 	var configMapsToUpdate []*apiv1.ConfigMap
-	if updateallowedcontext.IsUpdateAllowed(ctx) {
+	{
```

Comment:
With this PR we are going to update the config maps. This is a first step towards enabling updates for KVM guest clusters. This implies cloud configs are going to be reconciled as soon as they change.